streams/src/test/java/org/apache/kafka/test/ProcessorTopologyTestDriver.java

```diff
@@ -242,7 +242,7 @@ public ProcessorTopologyTestDriver(final StreamsConfig config,
      * @param value the raw message value
      * @param timestamp the raw message timestamp
      */
-    private void process(final String topicName,
+    public void process(final String topicName,
```

Comment:
why switch from `private`? Seems like it doesn't need to be `public`

Reply:
@bbejeck true, I don't use this version of the method in the newly created test. But I thought it would be nice to be consistent. So what we now have is 2 pairs:

```
public void process(
  final String topicName,
  final byte[] key,
  final byte[] value)
  
public void process(
  final String topicName, 
  final byte[] key, 
  final byte[] value,
  final long timestamp)
```
and
```
public <K, V> void process(
  final String topicName,
  final K key,
  final V value,
  final Serializer<K> keySerializer,
  final Serializer<V> valueSerializer)

public <K, V> void process(
  final String topicName,
  final K key,
  final V value,
  final Serializer<K> keySerializer,
  final Serializer<V> valueSerializer,
  final long timestamp)
```